src/internal_modules/roc_core/list.h

```diff
@@ -147,6 +147,36 @@ class List : public NonCopyable<> {
         insert_(element, NULL);
     }
 
+    //! Pop first element from list
+    //!
+    //! @remarks
+    //!  - removes first element of list
+    //!  - releases ownership of removed element
+    //!
+    //! @pre
+    //!  the list should not be empty.
+
+    void pop_front() {
```

Comment:
nitpick: please remove empty line between comment and method.

```suggestion
    //!  the list should not be empty.
    void pop_front() {
```